build.sbt

```diff
@@ -39,10 +39,9 @@ ThisBuild / githubWorkflowAddedJobs ++= Seq(
       WorkflowStep.Run(
         List("cd scalafix", "sbt ci"),
```

Comment:
Still, this fast-running job is not particularly helpful for our cache, since it doesn't use the primary `build.sbt`. This is something to fix in sbt-typelevel.
- https://github.com/typelevel/sbt-typelevel/issues/268